WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogDetailViewModel.kt

```diff
@@ -95,8 +89,19 @@ class ActivityLogDetailViewModel
         rewindStatusService.stop()
     }
 
-    private fun onRewindClicked() {
-        _item.value?.let { _showRewindDialog.postValue(it) }
+    fun onRangeClicked(activity: FragmentActivity, range: FormattableRange) {
+        formattableContentClickHandler.onClick(activity, range)
+    }
+
+    fun onRewindClicked(model: ActivityLogDetailModel) {
+        if (model.rewindId != null) {
+            _showRewindDialog.value = model
+        } else {
+            AppLog.e(
```

Comment:
We may consider displaying a Toast message. Wdyt?

Reply:
This should never happen because there is flag `rewindable` which we use to show/hide the rewind button and when it's set to `true`, the `rewindId` should always be there. However, since we have it in two fields, I added this logging to see if we can trust the API.

Reply:
👍